Sources/Segment/Timeline.swift

```diff
@@ -210,18 +210,28 @@ extension DestinationPlugin {
         }
         return result
     }
+    
+    internal func isDestinationEnabled(integrations: JSON?) -> Bool {
```

Comment:
maybe we should change the input parameter to `RawEvent` so that the api isnt misused with any kind of JSON